vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -41,6 +42,12 @@ const uint64_t kSustainAdNotificationInteractionAfterSeconds = 10;
 const uint64_t kDefaultCatalogPing = 2 * base::Time::kSecondsPerHour;
 const uint64_t kDebugCatalogPing = 15 * base::Time::kSecondsPerMinute;
 
+const char kCountrySubdivisionAllowedForRegion[] = "US";
```

Comment:
This feels like it should be an array since it's possible we'll extend it to other countries at some point. Also it would fit better with `kValidSubdivisionCountryCodes` which already supports the ability to add other countries.

Reply:
yeah I think that makes sense. I had the US flag before implementing the whitelist and didn't consolidate that part of the code to. Will change to be more future-proof

Reply:
changed, gh didn't recalculate
https://github.com/brave/brave-core/pull/5602/files#diff-3376a68e0782abbaa02acebe55c40f54R14 and
https://github.com/brave/brave-core/pull/5602/files#diff-c33be94bc446f2eda08f8ce0383f81b2R41